karma.conf.js

```diff
@@ -0,0 +1,106 @@
+module.exports = function(config) {
+
+  'use strict';
```

Comment:
The `use strict`? When i move it to the top outside the module.exports i get a lint error `Use the function form of 'use strict'`. Let me know if I'm missing something.